code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { BackButton } from '../components/BackButton'
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [movie, setMovie] = useState() 
+
+  const MOVIE_DETAILS_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=e20be4c51fce8750763fdb57c2e46728&language=en-US](https://api.themoviedb.org/3/movie/%7Bmovie_id%7D?api_key=%7Bapi_key%7D&language=en-US)`
+
+useEffect(() => {
+  fetch(MOVIE_DETAILS_URL)
+  .then (response => response.json())
+  .then (json => setMovie(json))
+  .catch(err => console.error(err))
```

Comment:
Please remember to remove all console outputs from your code before submitting. 👮🏻

Reply:
Hi! we have now removed the console output 🐧